03week/airplane/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <link rel="stylesheet" href="css/style.css">
+    <meta charset="utf-8">
+    <title>Airline</title>
+  </head>
+  <body>
+    <div class="header">
+      <img src="./img/logo.png" alt="me" class="size"/>
+    </div>
+    <div class ="nav">
+      <a href="http://www.google.com">
+        <img src="./img/home-icon.png" alt="hm"/>&ensp;Home &ensp;</a>
+      <a href="https://www.facebook.com/">
+        <img src="./img/deal-icon.png" alt="deal"/>&ensp;Travel Deals&ensp;</a>
+      <a href="https://www.youtube.com/">
+        <img src="./img/schedule-icon.png" alt="clock">&ensp;Flight Schedule&ensp;</a>
+      <a href="https://www.amazon.com/">
+        <img src="./img/login-icon.png" alt="deal"/>&ensp;Register/Login</a>
+    </div>
```

Comment:
Let's try not to force the browser to add blank spaces. we can do that with our css!